src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp

```diff
@@ -689,16 +689,44 @@ PYBIND11_MODULE(simplnx, mod)
   py::class_<RectGridGeom, IGridGeometry, std::shared_ptr<RectGridGeom>> rectGridGeom(mod, "RectGridGeom");
 
   py::class_<INodeGeometry0D, IGeometry, std::shared_ptr<INodeGeometry0D>> iNodeGeometry0D(mod, "INodeGeometry0D");
+  iNodeGeometry0D.def(
+      "resize_vertices",
+      [](INodeGeometry0D& nodeGeometry0D, int32 size) {
```

Comment:
I tested with `usize` here and didn't have any issues. Do you have the code you were running and any error messages?